docs/contributing.md

```diff
@@ -1,16 +1,33 @@
 # Contributing
-Contributions are welcome. We have a few requests of any new contributors.
+Contributions are welcome. We have a few requests for new contributors:
 
-* Follow styling as designated in our [styling documentation](/docs/style.md).
-* Ensure that all changes which affect logic are covered by unit tests. 
-* Do not introduce any unit test failures/regressions.
-* Turn on automated github actions in your fork to have github run all the unit tests after pushing. See example below:
+* **Follow styling guidelines.**
+  Please take a look at the [code style documentation](/docs/style.md)
+  to ensure uniformity and ease of communication.
```

Comment:
Minor: This text is indented by two spaces when the following bullet point aren't. Note in either case, Markdown will combine the following lines with the above bullet point into one paragraph / list item. I assume this is intentional.

Reply:
yeah, this what i meant by

> The other bullet points are also missing proper indentation.

The other listings should be fixed as well.